rest/src/main/kotlin/builder/component/SelectMenuBuilder.kt

```diff
@@ -45,6 +47,8 @@ public class SelectMenuBuilder(
     /**
      * Adds a new option to the select menu with the given [label] and [value] and configured by the [builder].
      *
+     * **Available only in [ComponentType.StringSelect]**
```

Comment:
I think this should be deprecated instead and be added to the new `StringSelectBuilder` subtype.

Reply:
How can I override an inline fun?

Reply:
I've got a solution that uses a hack with `kotlin.internal.LowPriorityInOverloadResolution`, I'll push it later if that's fine for you.

Reply:
No problem, thank you!

Reply:
I've committed the solution here: https://github.com/kordlib/kord/pull/707/commits/b9a0682e4359387a3c4c4b15730a32285b279218

Reply:
You can try out that the automatic replacement works as intended with code like:
```kotlin
import dev.kord.core.behavior.channel.createMessage
import dev.kord.rest.builder.message.create.actionRow

channel.createMessage {
    actionRow {
        stringSelect("custom-id") {
            option("label", "value") { description = "description" }
        }
    }
}
```
It will then add `dev.kord.rest.builder.component.option` to the imports. The extension function is used instead of the member because of `@kotlin.internal.LowPriorityInOverloadResolution`.